docs/docs/azure.md

```diff
@@ -41,28 +38,23 @@ question mark is the size. Other distributions should work as long as they are o
     * Pick a size that's got plenty of RAM. For example, you might want to click "Change size" to find `DC4s_v3` type
     * Ensure that the public inbound ports are open for SSH access
 
-    ![](images/create-d4sv3-vm.png)
+    ![](images/DC4S_V3 Screenshot.png)
     ![](images/create-d4sv3-vm-publicinbound.png)
 
 After you have logged on to the VM:
 
-1. Check that the `enclave` device is present in the `/dev/sgx/` or `/dev/sgx_*` directory
-2. Check driver version `dmesg | grep sgx`. Conclave requires driver version 1.33+
-3. If either check fails:
-    * Download the [driver](https://download.01.org/intel-sgx/latest/dcap-latest/linux/distro/ubuntu20.04-server/)
-    * Follow the [install instructions](https://download.01.org/intel-sgx/sgx-dcap/1.8/linux/docs/Intel_SGX_DCAP_Linux_SW_Installation_Guide.pdf)
-4. Add your user into `sgx_prv` group to give it access to SGX.
+1. Check that the `enclave` device is present in the `/dev/sgx/` directory.
+2. Add your user into `sgx_prv` group to give it access to SGX.
 
 ```sh
 sudo usermod -aG sgx_prv $USER
 ```
-5. Log out from the VM and log in again.
+3. Log out from the VM and log in again.
 
 You have set up an Azure VM with the latest attestation protocols.
 
-You can skip the rest of this document if you are only looking for a quick set up.
 
-## Background: The DCAP protocol
+## The DCAP protocol
```

Comment:
This section is no longer relevant since Azure has everything setup. Can we move this to a separate page for now.

The one piece of information developers will need to know when setting up an SGX VM on Azure. and which should be mentioned in this page, is that they will need to use DCAP for attestation. So that means from a code point of view, they will need to use the `AttestationParameters.DCAP()` class when starting the enclave via `EnclaveHost.start`.

Reply:
Moved the section to a new page 'dcap.md'. Added a short instruction to use use the `AttestationParameters.DCAP()` class when starting the enclave via `EnclaveHost.start` for DCAP.